contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/__tests__/plugins/image-upload/image-html-to-md.spec.js

```diff
@@ -3,22 +3,12 @@ import imageHtmlToMd from '../../../plugins/image-upload/image-html-to-md';
 describe('imageHtmlToMd', () => {
   it('converts images html to markdown', () => {
     const input = `
-      First image: <img
-        alt=""
-        src="/content/storage/c/h/checksum.ext"
-        width="100"
-        height="200"
-        class="image-field"
-        data-checksum="checksum"
-      />
-      Second image: <img
-        alt="Second image"
-        src="/content/storage/9/4/94ffaf.png"
-        width="auto"
-        height="auto"
-        class="image-field"
-        data-checksum="94ffaf"
-      />`;
+      First image: <span is='markdown-image-field'>
+        ![](\${☣ CONTENTSTORAGE}/checksum.ext =100x200)
```

Comment:
Instead of passing parameters to the `markdown-image-field`, we just pass the image markdown.

This is actually kind of a workaround for an issue where `TUI.editor` likes to delete empty `spans`... but it also makes for a pretty nice API.